buildenv/jenkins/testJobTemplate

```diff
@@ -110,6 +110,11 @@ ARCH_OS_LIST.each { ARCH_OS ->
 					BUILD_LIST = GROUP_BUILD_LIST_MAP[GROUP]
 				}
 
+				// for all jck builds, we need -Xfuture option passed in
+				if (GROUP == "jck") {
+					EXTRA_OPTIONS = "-Xfuture"
+				}
+				
```

Comment:
`EXTRA_OPTIONS ` needs to be set in the `pipelineJob`

Reply:
Right! Missed it. Added!